generic/release-checklist.md

```diff
@@ -0,0 +1,103 @@
+---
+# Template generated by https://github.com/coreos/repo-templates; do not edit downstream
+---
+
+# Release process
+
+The release process follows the usual PR-and-review flow, allowing an external reviewer to have a final check before publishing.
+
+## Requirements
+
+This guide requires:
+
+ * A web browser (and network connectivity)
+ * `git`
+ * [GPG setup][GPG setup] and personal key for signing
+ * Write access to this GitHub project
+
+## Release checklist
```

Comment:
There's a fair amount of duplication between this and other checklists.  Tera has an `include` tag that we could use, but I'm not sure tmpl8 is currently loading templates in a way that will allow this.  And I'm not sure it'd be a net win for maintainability anyway.  So this approach seems okay for now.

Reply:
I've rewritten the procedure to use a manual tag + PR workflow analog to other ones but sufficiently different that for now I'd say we should keep copies to make things simpler.